client/src/components/Header.css

```diff
@@ -0,0 +1,46 @@
+.Header {
+  display: grid;
+  grid-template-columns: [logo] 60px [navigation] auto;
+  padding: 20px;
+}
+
+.Header__navigation__wrapper {
```

Comment:
I'd make this position: absolute so it doesn't push down the content. Take a look at the pic:
![image](https://user-images.githubusercontent.com/6980777/130096126-38a881f4-7e95-4383-a3c5-c4f2c1073c01.png)